client/src/components/AppointmentCreateForm.js

```diff
@@ -12,14 +12,7 @@ import { useHistory } from "react-router-dom";
 const CREATE_APPOINTMENT = gql`
   mutation createAppointment($startTime:Int! $endTime:Int! $address:String! $note:String! ){
     createAppointment(startTime:$startTime, endTime:$endTime, address:$address, note:$note){
-      id,
-      location{
-        id
-        address
-        note
-      },
-      startTime,
-      endTime
+     startTime
```

Comment:
what does the warning say? Asking for startTime seems strange to me. If you must return a field, return ID.

Reply:
Never Mind. I just retried to delete all of the field names for return. It did not show me any warnings this time. Maybe something wrong happened when I tried it for the first time.